scripts/crosschain-token-transfer.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+require('dotenv').config();
+
+const {
+  Contract,
+  Wallet,
+  providers: { JsonRpcProvider },
+} = require('ethers');
+
+const {
+  printLog,
+  printObj,
+} = require('./logging');
+
+const path = require('node:path');
+
+// these environment variables should be defined in an '.env' file
+const contractsPath = process.env.CONTRACTS_PATH;
+const url = process.env.URL;
+const privKey = process.env.PRIVATE_KEY;
+const destinationChain = process.env.DESTINATION_CHAIN;
+const symbol = process.env.SYMBOL;
+const amount = process.env.AMOUNT;
+const gatewayAddress = process.env.GATEWAY_ADDRESS;
+
+// the ABIs for the contracts below must be manually downloaded/compiled
+const IAxelarGateway = require(path.join(contractsPath,'IAxelarGateway.json'));
+const IERC20 = require(path.join(contractsPath,'IERC20.json'));
+
+const provider = new JsonRpcProvider(url);
+const wallet = new Wallet(privKey, provider);
+const gateway = new Contract(gatewayAddress, IAxelarGateway.abi, wallet);
+const payload = Buffer.from([]);
+let transactions = {};
+
+printLog(`approving amount of ${amount}${symbol}`);
+
+(async () => {
+  const tokenAddress = await gateway.tokenAddresses(symbol);
+  const token = new Contract(tokenAddress, IERC20.abi, wallet);
+  return token.approve(gatewayAddress, amount);
+})()
+.then((tx) => {
+  tx.wait();
```

Comment:
This does not work cuz it returns a promise. 
```suggestion
  await tx.wait();
```

Reply:
it doesn't work

```
joaosousa@MBP-de-Joao axelar-cgp-solidity % node scripts/crosschain-token-transfer.js
/Users/joaosousa/go/src/github.com/axelarnetwork/axelar-cgp-solidity/scripts/crosschain-token-transfer.js:60
  await tx.wait();
  ^^^^^

SyntaxError: await is only valid in async functions and the top level bodies of modules
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1032:15)
    at Module._compile (node:internal/modules/cjs/loader:1067:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1155:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47

```

Reply:
I added the `async` modifier to the anonymous function and it looks to have fixed the issue